torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -127,6 +130,48 @@ RegisterOperators reg(
            };
          },
          aliasAnalysisSpecialCase()),
+     Operator(
+         prim::AllocateStorage, /*size, device*/
+         [](const Node* node) -> Operation {
+           size_t total_size = node->i(attr::total_size);
+           auto device = static_cast<DeviceType>(node->i(attr::device));
+           return [total_size, device](Stack* stack) {
+             at::DataPtr buffer =
+                 MemoryPlanner::allocate_buffer(total_size, device);
+             auto storage = c10::Storage(
+                 c10::Storage::use_byte_size_t(),
+                 total_size,
+                 std::move(buffer),
+                 /*allocator=*/nullptr,
+                 /*resizable=*/false);
+             push(stack, std::move(storage));
+           };
+         },
+         aliasAnalysisSpecialCase()),
```

Comment:
all these will need special cases in Alias Analysis later on. Unless we are planning on doing this as the very last optimization